CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Version changelog
 
+## 0.17.0
+
+* Use covariant type for `@retried(on=[...])` ([#486](https://github.com/databricks/databricks-sdk-py/pull/486)).
+* Configure request timeout using existing parameter from Config ([#489](https://github.com/databricks/databricks-sdk-py/pull/489)).
+* Make contents of `__init__.py` equal across projects ([#488](https://github.com/databricks/databricks-sdk-py/pull/488)).
+* Update SDK to Latest OpenAPI Specification ([#501](https://github.com/databricks/databricks-sdk-py/pull/501)).
+
+Note: This release contains breaking changes, please see below for more details.
```

Comment:
Highlight for breaking change. Clubber similar types of changes in changelog manually for better readability.